pkg/operation/cloudbotanist/openstackbotanist/addons.go

```diff
@@ -50,5 +50,11 @@ func (b *OpenStackBotanist) GenerateAdmissionControlConfig() (map[string]interfa
 
 // GenerateNginxIngressConfig generates values which are required to render the chart nginx-ingress properly.
 func (b *OpenStackBotanist) GenerateNginxIngressConfig() (map[string]interface{}, error) {
-	return common.GenerateAddonConfig(nil, b.Shoot.NginxIngressEnabled()), nil
```

Comment:
As transporting the `loadBalancerSourceRanges` to the Helm chart seems to be not cloud specific, please move this part to https://github.com/gardener/gardener/blob/master/pkg/operation/hybridbotanist/addons.go#L147 and keep the cloud specific implementations untouched.